pyoanda/client.py

```diff
@@ -121,6 +121,26 @@ def __call_stream(self, uri, params=None, method="get"):
         else:
             return resp
 
+    def get_events(self):
+        """
+            See more: http://developer.oanda.com/rest-live/streaming/#eventsStreaming
+            
+        """
+        url = "{0}/{1}/events".format(
+            self.domain_stream,
+            self.API_VERSION
+        )
+        params = {}
+
+        call = {"uri": url, "params": params, "method": "get"}
+
+        try:
+            return self._Client__call_stream(**call)
```

Comment:
Can we use parameters instead of dictionary.

Not big deal, it is just to keep the same style